object_list:
- arm_name: biscuits
  object_name: right
  pick_pose:
    orientation:
      w: 1.0
      x: 0.0
      y: 0.0
      z: 0.0
    position:
      x: 0.5417367219924927
      y: -0.24095962941646576
      z: 0.7097996473312378
  place_pose:
    orientation:
      w: 1.0
      x: 0.0
      y: 0.0
      z: 0.0
    position:
      x: 0.0
      y: -0.71
      z: 0.605
  test_scene_num: 1
- arm_name: soap
  object_name: right
  pick_pose:
    orientation:
      w: 1.0
      x: 0.0
      y: 0.0
      z: 0.0
    position:
      x: 0.5389411449432373
      y: -0.01819833368062973
      z: 0.6769629716873169
  place_pose:
    orientation:
      w: 1.0
      x: 0.0
      y: 0.0
      z: 0.0
    position:
      x: 0.0
      y: -0.71
      z: 0.605
  test_scene_num: 1
- arm_name: soap2
  object_name: left
  pick_pose:
    orientation:
      w: 1.0
      x: 0.0
      y: 0.0
      z: 0.0
    position:
      x: 0.44514018297195435
      y: 0.22380733489990234
      z: 0.6806973814964294
  place_pose:
    orientation:
      w: 1.0
      x: 0.0
      y: 0.0
      z: 0.0
    position:
      x: 0.0
      y: 0.71
      z: 0.605
  test_scene_num: 1
